src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const { id: tabId, content } = tabs.find(tab => tab.id === selectedTabId)
+    || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed" data-cy="tab-content">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={cn(
+                  { 'is-active': tabId === id },
+                )}
+                data-cy="Tab"
+                key={id}
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (selectedTabId !== id) {
+                      onTabSelected(tab);
+                    }
+                  }}
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here